ADAL/src/ADRequestParameters.m

```diff
@@ -98,23 +98,44 @@ - (void)setRedirectUri:(NSString *)redirectUri
     _redirectUri = [redirectUri msidTrimmedString];
 }
 
+- (void)setScope:(NSString *)scope
```

Comment:
Q: this refers to a space separated scopes string, correct? 
If so, 1. scope->scopes and 2. Perhaps make this a set?

Reply:
No reason to make it a set, because ADAL never does any operations on it (just passes it as a space separated string to ESTS). I agree on the name of the property though.